.changes/0832-eventpage-fixes.md

```diff
@@ -0,0 +1,2 @@
+- [fix] fixed My profile page widget tree errors 
+- [fix] fixed Events page card display errors 
```

Comment:
Remember the changelog entries are meant for _end_ users to understand. They don't understand what a "widget tree error" is ...

```suggestion
- [fix] Cases in which 'My profile' showed up blank 
- [fix] 'Events page' showing errors for specific events. 
```

Reply:
Noted ,  I'll be doing that .